exercises/two-fer/src/test/kotlin/TwoferTest.kt

```diff
@@ -0,0 +1,31 @@
+import org.junit.Ignore
+import org.junit.Test
+import kotlin.test.assertEquals
+
+
+class TwoferTest {
+
+    @Test
+    fun noNameGiven() {
+        assertEquals("One for you, one for me.", twofer())
+    }
+
+    @Test
+    @Ignore
+    fun emptyStringGiven() {
+        assertEquals("One for you, one for me.", twofer(""))
+    }
```

Comment:
Let's mimic the Java tests and (1) have the expected string here be "One for , one for me.", and (2) move this test to the end of the suite.